t_frang/test_concurrent_connections.py

```diff
@@ -241,15 +245,20 @@ def test_clear_client_connection_stats(self, name, clients_n: int, warning_n: in
         Check that Tempesta cleared client connection stats and
         new connections are established.
         """
-        self.set_frang_config(frang_config="concurrent_tcp_connections 2;\n\tip_block off;")
+        self.set_frang_config(frang_config="concurrent_tcp_connections 2;\n")
+        tempesta = self.get_tempesta()
 
         non_blocked_clients = [self.get_client(f"deproxy-{n}") for n in range(2)]
         blocked_clients = [self.get_client(f"deproxy-{n}") for n in range(2, clients_n)]
 
         for n in [warning_n, warning_n * 2]:
-            for client in non_blocked_clients + blocked_clients:  # establish 2 or more connections
+            for client in non_blocked_clients:
                 client.start()
-                client.make_request(client.create_request(method="GET", headers=[]))
+                client.wait_for_connection_open(strict=True)
+
+            for client in blocked_clients:  # establish 2 or more connections
+                client.start()
+                client.wait_for_connection_open()
```

Comment:
We expect the client to be blocked here
```suggestion
                self.assertIsNone(client.wait_for_connection_open())
```